synapse/lib/lmdbslab.py

```diff
@@ -310,6 +310,77 @@ def set(self, name: str, valu):
     def get(self, name: str, defv=0):
         return self.cache.get(name.encode(), defv)
 
+class LruHotCount(s_base.Base):
+    '''
+    HotCount with size limit and LRU cache for large key sets.
+    '''
+    EncFunc = staticmethod(s_common.signedint64en)
```

Comment:
snake case plz

Reply:
Could probably rename `encode` and `decode` as well :)

Reply:
updated to encode/decode